db/schema.rb

```diff
@@ -39,7 +39,7 @@
   create_table "documents", force: :cascade do |t|
     t.string "content_id", null: false
     t.string "locale", null: false
-    t.string "document_type"
+    t.string "document_type_id"
```

Comment:
should this field be null: false? (I know it's not introduced here mind)

Reply:
May as well :-). Done.